x-pack/plugins/security_solution/cypress/integration/timeline_template_creation.spec.ts

```diff
@@ -91,8 +84,6 @@ describe.skip('Timeline Templates', () => {
     cy.get(TIMELINE_TITLE).should('have.attr', 'value', timeline.title);
     cy.get(TIMELINE_DESCRIPTION).should('have.attr', 'value', timeline.description);
     cy.get(TIMELINE_QUERY).should('have.text', timeline.query);
-    // Comments this assertion until we agreed what to do with the filters.
-    // cy.get(TIMELINE_FILTER(timeline.filter)).should('exist');
```

Comment:
I think we should not delete these lines, since as soon as the filter bug is fixed we need to uncomment this to make sure we don't replicate the same bug again 😊